dataservice/api/task/__init__.py

```diff
@@ -0,0 +1,2 @@
+from dataservice.api.task.resources import TaskAPI
```

Comment:
Github won't let me put a comment on dataservice/api/task/README.md but i'll put my comment here. README has to be updated:
This
```
ID Prefix: `CT`
```
should be:
```
ID Prefix: <whatever prefix we decide>
```

Reply:
Same comment for dataservice/api/task_genomic_file/README.md